R/attach.R

```diff
@@ -0,0 +1,95 @@
+core <- c("crane", "gtsummary")
+
+core_unloaded <- function() {
+  search <- paste0("package:", core)
+  core[!search %in% search()]
+}
+
+# Attach the package from the same package library it was
+# loaded from before. https://github.com/tidyverse/tidyverse/issues/171
+same_library <- function(pkg) {
+  loc <- if (pkg %in% loadedNamespaces()) dirname(getNamespaceInfo(pkg, "path"))
+  library(pkg, lib.loc = loc, character.only = TRUE, warn.conflicts = FALSE)
+}
+
+crane_attach <- function() {
+  to_load <- core_unloaded()
+  if (!is_empty(to_load)) to_load <- union("crane", y = to_load)
+
+  suppressPackageStartupMessages(
+    lapply(to_load, same_library)
+  )
+
+  invisible(to_load)
+}
+
+crane_attach_message <- function(to_load) {
```

Comment:
to test

Reply:
I am not sure how to test without adding deps that would introduce a conflict. any ideas?

Reply:
Ahhh, do you have the bandwidth to add?

Reply:
I will do it today